lib/crypto/src/crypto.erl

```diff
@@ -642,87 +696,96 @@ nif_stub_error(Line) ->
 -ifndef(CRYPTO_VSN).
 -define(CRYPTO_VSN, "??").
 -endif.
+
 version() -> ?CRYPTO_VSN.
 
--spec start() -> ok | {error, Reason::term()}.
+-spec start() -> ok | {error, Reason :: term()}.
 start() ->
     case application:start(crypto) of
         ok ->
-            _ = supports(curves), % Build curves cache if needed
+            % Build curves cache if needed
```

Comment:
Is "have comments after code" a choice?

Reply:
Hm... if I understand correctly this is an opinionated formatter 😄 (saw this now on the GitHub page). I get it, but I didn't think it was 😢 I prefer "options". Since that's the case, I'd have to run it over my personal code base to make more informed decisions. Some it'll change stuff and I'll like it, some it'll change stuff and I won't. Is there a particular reason for it to not be configurable, in any case?

Reply:
We should probably create a separate issue, but the opinionated nature of `erlfmt` is being discussed here https://github.com/WhatsApp/erlfmt/issues/167

Please try it out on your own code and give us feedback, so we can try to make as best a decision as possible.